### PR TITLE
Doc: Realign release notes and add known issue

### PR DESCRIPTION
**Pull request by karenzone**
Adds these PRs to 8.16 branch (effectively a forwardport): 

- https://github.com/elastic/logstash/pull/16611
- https://github.com/elastic/logstash/pull/16661 (with minor formatting changes)

#### PREVIEWS
- https://logstash_bk_16663.docs-preview.app.elstc.co/guide/en/logstash/8.16/logstash-8-16-0.html
- https://logstash_bk_16663.docs-preview.app.elstc.co/guide/en/logstash/8.16/logstash-8-15-4.html
- https://logstash_bk_16663.docs-preview.app.elstc.co/guide/en/logstash/8.16/logstash-8-15-3.html
- https://logstash_bk_16663.docs-preview.app.elstc.co/guide/en/logstash/8.16/logstash-8-15-2.html
- https://logstash_bk_16663.docs-preview.app.elstc.co/guide/en/logstash/8.16/logstash-8-15-1.html
- https://logstash_bk_16663.docs-preview.app.elstc.co/guide/en/logstash/8.16/logstash-8-15-0.html

### CODE REVIEW

**Comment by robbavey** [comment]
This will also need https://github.com/elastic/logstash/pull/16664 🤦

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -72,6 +73,15 @@ This section summarizes the changes in the following releases:
 [[logstash-8-16-0]]
 === Logstash 8.16.0 Release Notes
 
+[[known-issues-8-16-0]]
+==== Known issue
+
+**{ls} may fail to shut down under some circumstances when you are using <<pipeline-to-pipeline>>.**
+Check out issue https://github.com/elastic/logstash/issues/16657[#16657] for details.
+
+Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.properties`.
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:82]:
```suggestion
Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.options`.
```

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -155,9 +165,61 @@ the event or by sending it to the pipeline's DLQ if enabled. https://github.com/
...
+[[logstash-8-15-4]]
+=== Logstash 8.15.4 Release Notes
+
+[[known-issues-8-15-4]]
+==== Known issue
+
+**{ls} may fail to shut down under some circumstances when you are using <<pipeline-to-pipeline>>.**
+Check out issue https://github.com/elastic/logstash/issues/16657[#16657] for details.
+
+Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.properties`.
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:177]:
```suggestion
Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.options`.
```

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -155,9 +165,61 @@ the event or by sending it to the pipeline's DLQ if enabled. https://github.com/
...
 [[logstash-8-15-3]]
 === Logstash 8.15.3 Release Notes
 
+[[known-issues-8-15-3]]
+==== Known issue
+
+**{ls} may fail to shut down under some circumstances when you are using <<pipeline-to-pipeline>>.**
+Check out issue https://github.com/elastic/logstash/issues/16657[#16657] for details.
+
+Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.properties`.
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:219]:
```suggestion
Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.options`.
```

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -185,6 +247,16 @@ the event or by sending it to the pipeline's DLQ if enabled. https://github.com/
 [[logstash-8-15-2]]
 === Logstash 8.15.2 Release Notes
 
+[[known-issues-8-15-2]]
+==== Known issue
+
+**{ls} may fail to shut down under some circumstances when you are using <<pipeline-to-pipeline>>.**
+Check out issue https://github.com/elastic/logstash/issues/16657[#16657] for details.
+
+Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.properties`.
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:256]:
```suggestion
Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.options`.
```

File: docs/static/releasenotes.asciidoc [hunk 5]
```diff
@@ -226,6 +298,12 @@ Check out issue https://github.com/elastic/logstash/issues/16437[#16437] for det
 +
 Workaround: Downgrade to {ls} 8.15.0, or temporarily avoid using environment and keystore variable references.
 
+* **{ls} may fail to shut down under some circumstances when you are using <<pipeline-to-pipeline>>.**
+Check out issue https://github.com/elastic/logstash/issues/16657[#16657] for details.
++
+Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.properties`.
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:304]:
```suggestion
Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.options`.
```

File: docs/static/releasenotes.asciidoc [hunk 6]
```diff
@@ -259,6 +337,15 @@ Workaround: Downgrade to {ls} 8.15.0, or temporarily avoid using environment and
 [[logstash-8-15-0]]
 === Logstash 8.15.0 Release Notes
 
+[[known-issues-8-15-0]]
+==== Known issue
+
+**{ls} may fail to shut down under some circumstances when you are using <<pipeline-to-pipeline>>.**
+Check out issue https://github.com/elastic/logstash/issues/16657[#16657] for details.
+
+Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.properties`.
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:346]:
```suggestion
Workaround: Add `-Dlogstash.pipelinebus.implementation=v1` to `config/jvm.options`.
```

**Review by robbavey — APPROVED** [review]
LGTM

**Comment by karenzone** [comment]
17097 (or sooner)

**Comment by karenzone** [comment]
@logstashmachine backport 8.x

**Pull request merged**